utils/test_utils/src/chain_structures.rs

```diff
@@ -0,0 +1,220 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use address::Address;
+use base64;
+use blocks::{
+    Block, BlockHeader, EPostProof, EPostTicket, FullTipset, Ticket, TipSetKeys, Tipset, TxMeta,
+};
+use chain::TipSetMetadata;
+use cid::{multihash::Blake2b256, Cid};
+use crypto::{Signature, Signer, VRFResult};
+use encoding::to_vec;
+use forest_libp2p::blocksync::{BlockSyncResponse, TipSetBundle};
+use forest_libp2p::rpc::RPCResponse;
+
+use message::{SignedMessage, UnsignedMessage};
+use num_bigint::BigUint;
+use std::error::Error;
```

Comment:
This doesn't compile by default, could just add the `#![cfg(feature = "test_constructors")]` tag at the top of this file and may also have to add `#[cfg(feature = "test_constructors")]` to the lib.rs reference to this module. No need to add the feature tag to everything specifically, so can remove those